src/jenkins.py

```diff
@@ -105,3 +192,139 @@ def unlock_jenkins(connectable_container: Container) -> None:
     version = get_version()
     connectable_container.push(LAST_EXEC_VERSION_PATH, version, encoding="utf-8", make_dirs=True)
     connectable_container.push(WIZARD_VERSION_PATH, version, encoding="utf-8", make_dirs=True)
+
+
+def _install_config(connectable_container: ops.Container) -> None:
+    """Install jenkins-config.xml.
+
+    Args:
+        connectable_container: The connectable Jenkins workload container.
+    """
+    with open("templates/jenkins-config.xml", encoding="utf-8") as jenkins_config_file:
+        connectable_container.push(CONFIG_FILE_PATH, jenkins_config_file)
+
+
+def _install_plugins(connectable_container: ops.Container) -> None:
+    """Install Jenkins plugins.
+
+    Download Jenkins plugins. A restart is required for the changes to take effect.
+
+    Args:
+        connectable_container: The connectable Jenkins workload container.
+
+    Raises:
+        JenkinsPluginError: if an error occurred installing the plugin.
+    """
+    plugins = " ".join(set(REQUIRED_PLUGINS))
+    proc: ops.pebble.ExecProcess = connectable_container.exec(
+        [
+            "java",
+            "-jar",
+            "jenkins-plugin-manager-2.12.11.jar",
+            "-w",
+            "jenkins.war",
+            "-d",
+            str(PLUGINS_PATH),
+            "-p",
+            plugins,
+        ],
+        working_dir="/srv/jenkins/",
```

Comment:
Just checking :)
HOME_PATH?

Reply:
The home path is `/var/jenkins` and `/srv/jenkins` is where the .war files are at. Perhaps I should put this as a constant too! Thanks for the comment!